nixos/modules/services/networking/dropbox.nix

```diff
@@ -0,0 +1,73 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.dropbox;
+
+in
+
+{
+  ###### interface
+
+  options.services.dropbox = {
+
+    enable = mkEnableOption ''
+      Whether to enable the dropbox service globally.
+    '';
+
+    install = mkEnableOption ''
+      Whether to install a user service for the dropbox daemon. Once
+      the service is started, the Dropbox desktop app is available.
+
+      The service must be manually started for each user with
+      "systemctl --user start dropbox" or globally through
+      <varname>services.emacs.enable</varname>.
```

Comment:
I guess this is a copy-paste leftover, or since when does Dropbox use emacs as a backend? :D

Reply:
Fixed.